src/Data/Csv/Parser.hs

```diff
@@ -43,6 +52,18 @@ import Control.Applicative ((<$>), (*>), (<*), pure)
 import Data.Monoid (mappend, mempty)
 #endif
 
+-- | Whether to escape fields with double quotes (@"@).
+data Escaping
+    = Escape
+      -- ^ Parse possible escaped fields. The delimiter can appear in
+      -- fields surrounded by double quotes; to write a double quote
+      -- character inside an escaped field, you must use two double
+      -- quote characters (@""@).
+    | NoEscape
+      -- ^ Parse fields that are not escaped. The delimiter can
+      -- /never/ appear in fields, but double quote characters can.
+  deriving (Eq, Show)
```

Comment:
:+1: for using a `data` with two constructors instead of a bool, that makes it much more readable at the call site.

Reply:
Ah, I see this was @fatho’s idea :sweat_smile: